backend/src/main/java/ubb/courses/backend/dtos/courses/CourseDTO.java

```diff
@@ -0,0 +1,21 @@
+package ubb.courses.backend.dtos.courses;
+
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import ubb.courses.backend.dtos.BaseDTO;
+
+import javax.validation.constraints.NotNull;
+import java.util.Date;
+
+@Data
+@EqualsAndHashCode(callSuper = true)
+public class CourseDTO extends BaseDTO {
+    @NotNull
+    public String name;
+    @NotNull
+    public String description;
+
+    private Date createdDate;
+
+    private Date modifiedDate;
```

Comment:
Those properties are used only for Admin auditing, please remove them from the DTO